### PR TITLE
Docsite: add Community and Contributing guides

### PR DESCRIPTION
**Pull request by Andersson007**
Dear maintainers,
As a part of the [Consolidating Ansible discussion platforms initiative](https://forum.ansible.com/t/proposal-consolidating-ansible-discussion-platforms/6812), this PR adds the communication section template defined by the community to the README and the docsite. Similar PRs are now being raised across all the Ansible ecosystem projects.

Thank you

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @oraNod

**Comment by Andersson007** [comment]
ready for review

File: docs/community.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+* `Get Help <https://forum.ansible.com/c/help/6>`_: get help or help others. Please add appropriate tags if you start new discussions, for example `ansible-sign`.
+* `Posts tagged with 'ansible-sign' <https://forum.ansible.com/tag/ansible-sign>`_: subscribe to participate in project-related conversations.
+* `Bullhorn newsletter <https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn>`_: used to announce releases and important changes.
+* `Social Spaces <https://forum.ansible.com/c/chat/4>`_: gather and interact with fellow enthusiasts.
+* `News & Announcements <https://forum.ansible.com/c/news/5>`_: track project-wide announcements including social events.
+
+For more information on the forum navigation, see `Navigating the Ansible forum <https://forum.ansible.com/t/navigating-the-ansible-forum-tags-categories-and-concepts/39>`_ post.
```
Comment (oraNod) [inline docs/community.rst:26]:
```suggestion
See `Navigating the Ansible forum <https://forum.ansible.com/t/navigating-the-ansible-forum-tags-categories-and-concepts/39>`_ for some practical advice on finding your way around.
```

File: README.md [hunk 2]
```diff
@@ -17,3 +17,7 @@ Documentation can be found on [ansible-sign.readthedocs.io](https://ansible.read
 including a
 [rundown/tutorial](https://ansible.readthedocs.io/projects/sign/en/latest/rundown.html)
 explaining how to use the CLI for basic project signing and verification.
+
+## Community
+
+Need help or want to discuss the project? See our [Community guide](https://ansible.readthedocs.io/projects/sign/en/latest/community.html) to learn how to join the conversation.
```
Comment (oraNod) [inline README.md:23]:
```suggestion
Need help or want to discuss the project? See our [Community guide](https://ansible.readthedocs.io/projects/sign/en/latest/community.html) join the conversation.
```

File: docs/contributing.rst [hunk 3]
```diff
@@ -0,0 +1,9 @@
+============
+Contributing
+============
+
+.. note::
+
+  Need help or want to discuss the project? See the :ref:`Community guide<community>` to learn how to join the conversation!
```
Comment (oraNod) [inline docs/contributing.rst:7]:
```suggestion
  Need help or want to discuss the project? See the :ref:`Community guide<community>` join the conversation!
```

File: docs/index.rst [hunk 4]
```diff
@@ -5,13 +5,18 @@ ansible-sign
 This is the documentation for the **ansible-sign** utility used for signing and
 verifying Ansible content.
 
+.. note::
+
+  Need help or want to discuss the project? See the :ref:`Community guide<community>` to learn how to join the conversation!
```
Comment (oraNod) [inline docs/index.rst:10]:
```suggestion
  Need help or want to discuss the project? See the :ref:`Community guide<community>` to join the conversation!
```

**Comment by Andersson007** [comment]
@oraNod all the suggestions are committed, thanks, PTAL

File: docs/contributing.rst [hunk 5]
```diff
@@ -0,0 +1,9 @@
+============
+Contributing
+============
+
+.. note::
+
+  Need help or want to discuss the project? See the :ref:`Community guide<community>` to join the conversation!
```
Comment (oraNod) [inline docs/contributing.rst:7]:
```suggestion
   Need help or want to discuss the project? See the :ref:`Community guide<community>` to join the conversation!
```

**Comment by oraNod** [comment]
@Andersson007 Thanks. I added one other nit. It also looks like there's still one suggestion to commit or resolve.

**Comment by Andersson007** [comment]
> @Andersson007 Thanks. I added one other nit. It also looks like there's still one suggestion to commit or resolve.

@oraNod oh, missed it somehow, done, PTAL

File: docs/index.rst [hunk 6]
```diff
@@ -5,13 +5,18 @@ ansible-sign
 This is the documentation for the **ansible-sign** utility used for signing and
 verifying Ansible content.
 
+.. note::
+
+  Need help or want to discuss the project? See the :ref:`Community guide<community>` to join the conversation!
```
Comment (oraNod) [inline docs/index.rst:10]:
```suggestion
   Need help or want to discuss the project? See the :ref:`Community guide<community>` to join the conversation!
```

**Review by oraNod — APPROVED** [review]
Sorry @Andersson007 there was one more nit but it LGTM otherwise. Thanks.

**Comment by Andersson007** [comment]
> Sorry @Andersson007 there was one more nit but it LGTM otherwise. Thanks.

@oraNod , committed, thanks! PTAL

**Review by oraNod — APPROVED** [review]

**Review by gundalow — APPROVED** [review]

**Pull request merged**